application/event_sender/event_sender.py

```diff
@@ -26,6 +30,12 @@ def lambda_handler(event: Dict[str, Any], context: LambdaContext) -> None:
         "Received change request",
         extra={"change_request": body["change_payload"], "correlation_id": logger.get_correlation_id()},
     )
+    message_received = body["message_received"]
+
     change_request = ChangeRequest(body["change_payload"])
     change_request.post_change_request()
+    now_ms = time_ns() // 1000000
```

Comment:
Is this milliseconds or nanoseconds? As we are using milliseconds elsewhere is this a problem for the latency calculation.
https://www.geeksforgeeks.org/python-time-time_ns-method/